plugins/utils/hydra/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.neovim-plugin.mkNeovimPlugin config {
+    name = "hydra";
+    originalName = "hydra.nvim";
+    defaultPackage = pkgs.vimPlugins.hydra-nvim;
+
+    maintainers = [maintainers.GaetanLepage];
+
+    extraOptions = {
+      # A list of `Hydra` definitions
+      hydras = import ./hydras-option.nix {inherit lib helpers;};
```

Comment:
Is there a reason you tend to favor inheriting specific attrs over doing something like this:

```nix
{
  lib,
  helpers,
  config,
  pkgs,
  ...
} @attrs: {
  hydras = import ./hydras-option.nix attrs;
}
```

?

This is more a general question, not feedback on this PR.

Reply:
I find this way to be more explicit and less obfuscating, but it does not matter to much to me.